Source/IdentityApi/IdentityApi/Interfaces/IUserLocationProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using IdentityApi.Models;
+
+namespace IdentityApi.Interfaces
+{
+    public interface IUserLocationProvider
+    {
+        /// <summary>
+        /// Checks if ip adress is blacklisted 
+        /// </summary>
+        public Task<bool> IsIPLockedAsync(string ipAddress);
```

Comment:
Same as above, remember to describe what we are returning, as it is not obvious.